lib/services/git-service.ts

```diff
@@ -93,6 +81,34 @@ export class GitService implements IGitService {
                 });
         });
     }
+
+    private getNpmPackageVersion(templateName: string): Promise<any> {
```

Comment:
In my opinion, this function is misplaced and is better suited for the util module. @manoldonev @ADjenkov ?

Reply:
I'm OK with that.

Reply:
I am not sure util is a better place for it as it will not be used anywhere else. In fact in https://github.com/NativeScript/nativescript-starter-kits/pull/33 I proposed and @ADjenkov accepted to move the `getPageTemplatesBaseUrl(...)` method from util and make it a private page service method. I would instead consider extracting `pageExists` from util as well (and moving it to page service) as in general I don't believe such non-generic methods need to be exposed there (not sure they should be exposed at all)

@ADjenkov ?

Reply:
Or we can expose NpmService and move it there?

Reply:
No doubt it will be a cleaner solution, on the other hand, for the time being it is a single function that does interactions with the NPM API